parser-typechecker/src/Unison/Runtime/Builtin.hs

```diff
@@ -2586,7 +2586,7 @@ declareForeigns = do
     \(bytes :: Bytes.Bytes) -> pure $ X.readKeyFileFromMemory $ L.toStrict $ Bytes.toLazyByteString bytes
 
   declareForeign Tracked "Tls.encodePrivateKey" boxDirect . mkForeign $
-    \(privateKey :: X.PrivKey) -> pure $ Util.Text.pack $ show privateKey
+    \(privateKey :: X.PrivKey) -> pure $ Util.Text.toUtf8 $ Util.Text.pack $ show privateKey
```

Comment:
this makes it so that the return type of encodePrivateKey is at least correct, so you don't get runtime type errors